.github/workflows/package-version-check.yml

```diff
@@ -0,0 +1,30 @@
+name: Version Update Check
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
```

Comment:
👇 If changes were made to files in `src` but the package.json's version hasn't been updated, the CI pipeline will be red.